hack/generate-operator-bundle-contents.py

```diff
@@ -82,5 +87,6 @@
 
 csv_filename = pathlib.Path(args.output_dir) / \
     f'{args.name}.v{args.current_version}.clusterserviceversion.yaml'
+os.makedirs(os.path.dirname(csv_filename), exist_ok=True)
```

Comment:
what is this good for please? It's not obvious to me on Friday.... :)

Reply:
Basically when we pass the args.output_dir, if the directory isn't already created (for example, bundle/manifests), the script fails because it cannot find the path. This makes sure that if the path doesn't exist, it creates it before doing the same with the file.